BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -556,7 +558,8 @@ def get_ev_occupant_number(mkc_activity_schedules)
     end
 
     vehicle = @hpxml_bldg.vehicles[0]
-    hours_per_year = (vehicle.hours_per_week / 7) * UnitConversions.convert(1, 'yr', 'day')
+    hours_per_week = get_ev_hours_per_week(vehicle)
```

Comment:
Found a bug while working on all of this. `hours_per_week` is not a required input, so we need to default it when not provided.

I should see if we can apply defaults to the entire HPXML Building (or a copy of the object, if we need to prevent writing the defaults out to disk) to generically prevent issues like this in the future...